ios/src/CountlyReactNative.m

```diff
@@ -358,31 +375,41 @@ + (void)onNotification:(NSDictionary *)notification
 
 RCT_EXPORT_METHOD(setLocation:(NSArray*)arguments)
 {
-  NSString* countryCode = [arguments objectAtIndex:0];
-  NSString* city = [arguments objectAtIndex:1];
-  NSString* location = [arguments objectAtIndex:2];
-  NSString* IP = [arguments objectAtIndex:3];
+  NSString* city = [arguments objectAtIndex:0];
+  NSString* country = [arguments objectAtIndex:1];
```

Comment:
The order of values read is wrong, country code is first and city is second

Reply:
Oh, sorry for that.